singlestoredb/management/job.py

```diff
@@ -159,54 +161,54 @@ def from_dict(cls, obj: Dict[str, Any]) -> 'Schedule':
         out = cls(
             execution_interval_in_minutes=int(obj['executionIntervalInMinutes']),
             start_at=to_datetime(obj['startAt']),
-            mode=obj.get('mode'),
```

Comment:
This was actually fine before since `obj.get('mode')` will return `None` if the key doesn't exist. The only time you need to do the extra check is if you are going to cast the result to another type and you want to also allow `None` as a value (e.g., `bool(obj['mode']) if 'mode' in obj else None`).